tests/Common/TokensTest.php

```diff
@@ -211,6 +211,14 @@ public function testGetToken(): void
         $this->assertIsInt($firstLimit['value']);
         $this->assertEquals($firstLimit['name'], $limitKeys[0]);
 
+        $admin = $verifiedToken['admin'];
```

Comment:
spis bych to videl na samostatny test, kde to vyzkousis to preskakovani pres aplikacni tokeny. takhle `verifiedToken` je admin token (link na admina uz tam je), takze to je easy. Ale tricky je prave ta situace, kdy to musi preskakat pres par aplikacnich tokenu pres creatora

Reply:
pravda, přidal jsem test